tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -71,12 +73,24 @@ spec:
         iib_service_account_secret=$(jq -r '.fbc.iibServiceAccountSecret // "iib-service-account"' ${DATA_FILE})
         build_tags=$(jq '.fbc.buildTags // []' ${DATA_FILE})
         add_arches=$(jq '.fbc.addArches // []' ${DATA_FILE})
+        hotfix=$(jq '.fbc.hotfix // "false"' ${DATA_FILE})
         build_timeout_seconds=$(jq -r --arg build_timeout_seconds ${default_build_timeout_seconds} \
             '.fbc.buildTimeoutSeconds // $build_timeout_seconds' ${DATA_FILE})
         target_index=$(jq -r '.fbc.targetIndex' ${DATA_FILE})
         fbc_fragment=$(jq -cr '.components[0].containerImage' ${SNAPSHOT_PATH})
 
-        # saves the target index required to sign
+        if [ "${hotfix}" == "true" ]; then
+          timestamp_format=$(jq -r --arg default_timestamp_format "%s" \
+            '.fbc.timestampFormat // $default_timestamp_format' ${DATA_FILE})
```

Comment:
What's the point of using a named argument here if it's hardcoded to `"%s"`? Why not just use that directly?

Reply:
Because I saw in different places different formats of TS in tags. If someone for any reason decides that this is not the format we want we don't need to change the code to have it changed.

Reply:
fixed.

Reply:
I don't follow.

What I'm saying is that it doesn't make sense to pass `"%s"` as an argument when it's hardcoded.

Instead of
```sh
timestamp_format=$(jq -r --arg default_timestamp_format "%s" \
            '.fbc.timestampFormat // $default_timestamp_format' ${DATA_FILE})
```
I'm suggesting
```sh
timestamp_format=$(jq -r \
            '.fbc.timestampFormat // "%s"' ${DATA_FILE})
```

I don't understand how this makes it any different in terms of changing the default in the future. It's hardcoded in this task anyway.

Reply:
OK, cool, thanks.